dependencies.tsv

```diff
@@ -21,6 +21,7 @@ gopkg.in/juju/charm.v5	git	39463053128b672308c459958d00f260b58ce79e	2015-05-14T1
 gopkg.in/juju/charm.v6-unstable	git	69e524672748337e749acb16dbeefe3e80c946c0	2015-05-05T17:09:44Z
 gopkg.in/juju/charmstore.v4	git	3a87b423c3eeb105b82bba2f87d593d1b5763845	2015-05-14T14:17:49Z
 gopkg.in/macaroon-bakery.v0	git	9593b80b01ba04b519769d045dffd6abd827d2fd	2015-04-10T07:46:55Z
```

Comment:
I'm a little worried your depending on two versions of the same library, that's not right surely.